executor/runner/runner.go

```diff
@@ -104,7 +105,7 @@ func WithRuntime(ctx context.Context, m metrics.Reporter, rp RuntimeProvider, cf
 }
 
 // StartTask can be called once to start a task, by a given Runner
-func (r *Runner) StartTask(taskID string, titusInfo *titus.ContainerInfo, mem int64, cpu int64, gpu int64, disk uint64) error {
+func (r *Runner) StartTask(taskID string, titusInfo *titus.ContainerInfo, mem int64, cpu int64, gpu int64, disk uint64, network uint64) error {
```

Comment:
If we intend to keep this around much longer, IMHO we should replace ContainerInfo with v1.Pod, and the bits to extract mem / cpu / gpu / disk / network should happen *in* here. Nothing to do with this PR specifically, but there's a bunch of stuff that could be extracted at the pod level that's more straightforward, helping us remove stuff from containerinfo.